commands.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/mendersoftware/go-lib-micro/config"
 	"github.com/mendersoftware/go-lib-micro/identity"
 	"github.com/mendersoftware/go-lib-micro/log"
+	"github.com/mendersoftware/useradm/client/tenant"
```

Comment:
can you move this import to the block with other `github.com/mendersoftware/useradm/` imports?

Reply:
fixed